Beta_Integrations/GmailSingleUser/GmailSingleUser.yml

```
@@ -0,0 +1,209 @@
+category: Authentication
```

Change category to: `Messaging`

```suggestion
category: Messaging
```